auth_matrix/templates/admin/auth_matrix/includes/object_tools_items.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+{% if request.user.is_staff %}
+    {% if perms.auth.view_user or perms.accounts.view_user %}
```

Comment:
* I think the staff check is not needed, since we're doing a permission check already
* we'll need a different solution for the permission check based on what the actual user model is, see the remark in the python code